kolibri/core/notifications/test/test_api.py

```diff
@@ -1131,137 +1141,141 @@ def setUpTestData(cls):
         cls.facility = FacilityFactory.create()
         cls.superuser = create_superuser(cls.facility)
 
-        cls.user1 = FacilityUserFactory.create(facility=cls.facility)
-        cls.user2 = FacilityUserFactory.create(facility=cls.facility)
+    def setUp(self):
```

Comment:
Hrm, what was the need for this update? Setting up commonly used initial data in the `setUpTestData` classmethod helps tests to run considerably faster, so if we can keep this at all, we should.

Reply:
I'm pretty sure at some point it doesn't work with `setUpTestData` and that's why I changed it to the `setUp` method, but I just reverted it and it works perfectly lol haha. Done!

Reply:
I also updated the `NotificationsAPITestCase` class to use the `setUpTestData` instead of `setUp`.